src/Test/Diagnostics/TestingOnly_WaitingService.cs

```diff
@@ -118,10 +118,5 @@ public void EnableActiveTokenTracking(bool enable)
                 waiter.Value.TrackActiveTokens = enable;
             }
         }
-
-        public void PumpingWait(Task task)
```

Comment:
Sadly I had to add these methods back because closed soucre depends on them.  Will root those out later on as well.